app/views/layouts/application.html.erb

```diff
@@ -7,6 +7,7 @@
     <%= csrf_meta_tags %>
     <%= stylesheet_link_tag    'application', media: 'all' %>
     <%= javascript_include_tag 'application' %>
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
```

Comment:
We might need to make this a `stylesheet_link_tag`. This is the first thing I'd try if we have any issues with it not showing when we deploy to Heroku.

Reply:
I had a conversation with Ben about this and he is under the impression that a `stylesheet_link_tag` increases the likelihood of breaking heroku.